issue-wanted.cabal

```diff
@@ -111,20 +111,20 @@ library
                      , elm-street ^>= 0.0.1
                      , github == 0.22
                      , http-api-data ^>= 0.4
-                     , http-client ^>= 0.5.14
+                     , http-client ^>= 0.6
                      , http-client-tls ^>= 0.3.5.3
                      , http-types ^>= 0.12
                      , lens ^>= 4.17
                      , mtl ^>= 2.2.2
                      , postgresql-simple ^>= 0.6.1
-                     , postgresql-simple-named ^>= 0.0.0.0
+                     , postgresql-simple-named ^>= 0.0
```

Comment:
I would prefer to keep it as it is. I'm not very sure in PVP comparison operators so I prefer to restrict the lower bounds explicitly.

Reply:
Do you mean as in `^>= 0.0.2`?

Reply:
To be precise, it is `^>= 0.0.2.0`.

Reply:
Would that be equivalent to `== 0.0.2.0` if we assume that the package won't have more components? I noticed that some dependency constraints for packages that you own do have `==` rather than `^>=`, and I assumed it was precisely because you'd be able to and prefer to bump both, which would be necessary when fixating the constraint this much.

I'll update it to `^>= 0.0.2.0`. Always curious to learn your practices.

Reply:
It is not the same as the `==` operator. `==` means exactly that version. It differs from `^>=` because `^>= 1.2.0` desugars to `>= 1.2.0 && < 1.3` (by the second part of the PVP version component).  I prefer to specify the full version bounds because of some weird comparison rules (example `0.0` is greater than `0.0.0.0`). That's why I'm trying to avoid implicit versions where possible.

Reply:
Thanks for clarifying.